Funcky.Analyzer/Funcky.Analyzer.Test/Verifiers/VisualBasicAnalyzerVerifier`1+Test.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Microsoft.CodeAnalysis.VisualBasic.Testing;
+
+namespace Funcky.Analyzer.Test
+{
+    public static partial class VisualBasicAnalyzerVerifier<TAnalyzer>
```

Comment:
Do we need `VisualBasicAnalyzer` testing stuff?

Reply:
Autogenerated by template

Reply:
I'd remove it if we don't want to support visual basic

Reply:
Removed Code referencing to VisualBasic c0eb09753a0772bfca8d8c88780598ecdd93672c and b0aeb44c2797d173babfc6ce8402cc95e1137c12